README.rst

```diff
@@ -1,5 +1,5 @@
 =========
-Open Zaak
+OpenZaak - productiewaardige API's voor Zaakgericht Werken
```

Comment:
Ik zou de tagline niet in de heading opnemen.

Suggestie: Plaats het onder de `====` en tussen \* om het italic weer te geven.

Reply:
We gebruiken nu ook overal Open[spatie]Zaak. Geen blocker maar wellicht klap op geven voor consistente naam voering.